src/sdk/namespace_fs.js

```diff
@@ -1964,69 +1974,81 @@ class NamespaceFS {
                 throw new S3Error(S3Error.InvalidObjectStorageClass);
             }
 
-            // Total 8 states
+            const now = new Date();
+
+            // Total 6 states (restore_request - 2, restore_expiry - 3)
             const restore_request = stat.xattr[GlacierBackend.XATTR_RESTORE_REQUEST] ?
                 parseInt(stat.xattr[GlacierBackend.XATTR_RESTORE_REQUEST], 10) : null;
-            const restore_ongoing = stat.xattr[GlacierBackend.XATTR_RESTORE_ONGOING] === 'true';
             const restore_expiry = stat.xattr[GlacierBackend.XATTR_RESTORE_EXPIRY] ?
                 new Date(stat.xattr[GlacierBackend.XATTR_RESTORE_EXPIRY]) : null;
```

Comment:
How do we handle bad date format, say if someone edits this xattr manually, which is also relevant for restore_request if it has a bad number - lets just be clear on how this behaves so that we can explain it. Perhaps it's best to log the issue and consider it as undefined?